.pre-commit-config.yaml

```diff
@@ -6,17 +6,19 @@ repos:
     hooks:
       # TODO: - id: trailing-whitespace
       # TODO: - id: end-of-file-fixer
```

Comment:
@dietmarkuehl should we use this checks too?

Reply:
seems fine. I think one of the editors I use likes to add a newline at the end while another may not. Getting that consistent seems good.

Reply:
ok, done